### PR TITLE
removes Gathering section and header

### PR DESCRIPTION
**Pull request by wgordon17**
Gathering has been completed, so the data is being removed from the site until our next Gathering

Signed-off by: Will Gordon <wgordon@redhat.com>

### CODE REVIEW

File: source/layouts/_navbar.erb [hunk 1]
```diff
@@ -16,9 +16,6 @@
             <a href="/briefings.html">Events &amp; Briefings</a>
           </li>
           <li>
-            <a href="/gathering">Gathering</a>
-          </li>
-          <li>
```
Comment (luciddreamz) [inline source/layouts/_navbar.erb]: Why did you remove this list item instead of the one on line 18?

Comment (luciddreamz) [inline source/layouts/_navbar.erb]: (j/k btw looks good)

Comment (wgordon17) [inline source/layouts/_navbar.erb]: Wait...did I really screw that up? Good lord, I must've briefly lost my mind!
